### DISCUSSION TITLE
MAUI Support is upcoming!

### DISCUSSION THREAD

**Discussion by enisn** [enisn/Xamarin.Forms.InputKit#224] [Announcements]
MAUI will be supported with `v4.0.0``

---

https://github.com/enisn/Xamarin.Forms.InputKit/issues/221

**Comment by enisn** [comment]
MAUI Package has been released with a new package name: [InputKit.Maui](https://www.nuget.org/packages/InputKit.Maui/)

**Reply by xiaohongSunLN** [reply]
Hi Sir, 
Thanks your support on MAUI. In my MAUI project I want to use the AdvancedEntry and Dropdown, But I didn't find the implement in SandboxMAUI, so I write the test code in my project, but the two controls can't work well. no act response of  Dropdown the list menu didn't display. can you update the support for  the AdvancedEntry and Dropdown. Thanks a lot!

**Reply by enisn** [reply]
Hi @xiaohongSunLN 

Unfortunately, the Dropdown control hasn't been implemented yet.
You can follow the progress from here: https://github.com/enisn/Xamarin.Forms.InputKit/issues/221

But, **AdvancedEntry** should work correctly. What kind of problem are you facing?

**Reply by xiaohongSunLN** [reply]
Hi enisn,
In my MacBook Pro, OS: macOS Monterey, version 12.5, 
I ran the project of SandboxMAUI, in the mainpage I click the "Advanced Entry & FormView" button,
there is nothing appeared.

In iPad simulator(15.5), click "Advanced Entry & FormView" button, app will crash and exit.

Can you give me some support for the issues. 
Thanks very much!

**Reply by enisn** [reply]
I'll check this case as soon as possible

**Reply by enisn** [reply]
I reproduced the issue, it's all about demo application, no need to release another version of InputIKit

Following PR solves the problem:
https://github.com/enisn/Xamarin.Forms.InputKit/pull/267

**Reply by enisn** [reply]
<img width="362" alt="image" src="https://user-images.githubusercontent.com/23705418/185451500-1886ade8-a0ae-4553-a667-1c409bfb9142.png">
it works well now
